pootle/settings/91-travis.conf

```diff
@@ -16,28 +16,31 @@ if os.environ.get("TRAVIS"):
     DATABASE_BACKEND = os.environ.get("DB")
     DATABASES = {
         'default': {
-            'ENGINE': 'django.db.backends.sqlite3',
-            'NAME': working_path('dbs/zing_travis.db'),
-            'USER': '',
+            'ENGINE': 'django.db.backends.mysql',
+            'NAME': "zing",
+            'USER': 'travis',
             'PASSWORD': '',
             'HOST': '',
             'PORT': '',
             'ATOMIC_REQUESTS': True,
+            'CHARSET':'utf8mb4',
+            'COLLATION':'utf8mb4_unicode_ci',
             'TEST': {
-                'NAME': '',
-                'CHARSET': 'utf8',
+                'NAME': 'test_zing',
+                'CHARSET':'utf8mb4',
+                'COLLATION':'utf8mb4_unicode_ci'
+            },
+            "OPTIONS":{
+                'init_command': "SET sql_mode='STRICT_ALL_TABLES'",
             }
         }
     }
 
-    if DATABASE_BACKEND.startswith("mysql"):
-        DATABASES['default']['ENGINE'] = 'django.db.backends.mysql'
-        DATABASES['default']['NAME'] = 'zing'
+    if DATABASE_BACKEND.startswith("sqlite"):
```

Comment:
Can you please explain this change? I think with this PR we're moving away from sqlite.

Reply:
It's to maintain sqlite support for user's that are on it. Thought I should keep it for compatibility